prisma/schema.prisma

```diff
@@ -0,0 +1,62 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:team.db"
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model Member {
+  slug              String   @id
+  name              String   @unique
+  email             String   @unique
+  title             String
+  slack             String?
+  github            String?
+  startDate         DateTime
+  headshot          String?
+  preferredPronouns String?
+  introEmail        String?
+  roleText          String?
+
+  orgs     Organization[]
+  teams    Team[]
+  subteams Subteam[]
+
+  location     Location? @relation(fields: [locationSlug], references: [slug])
+  locationSlug String?
+
+  manager     Member?  @relation("MemberToMember", fields: [managerSlug], references: [slug])
+  reports     Member[] @relation("MemberToMember")
+  managerSlug String?
```

Comment:
These are required for a self referring one to many. It looks pretty complicated to grok, but the editor will literally write all this for you after you write the `manager  Member?` part. It's pretty amazing.